src/EventHandler.cpp

```diff
@@ -25,34 +25,36 @@ const int MAX_MESSAGE_PROCESSING = 100;	/**
 										 * \todo	Make this configurable?
 										 */
 
-EventHandler::KeyModifier& operator|=(EventHandler::KeyModifier& a, const EventHandler::KeyModifier& b) noexcept
-{
-	using underlying = std::underlying_type_t<EventHandler::KeyModifier>;
-	auto underlying_a = static_cast<underlying>(a);
-	auto underlying_b = static_cast<underlying>(b);
-	a = static_cast<EventHandler::KeyModifier>(underlying_a | underlying_b);
-	return a;
-}
+namespace NAS2D {
+	EventHandler::KeyModifier& operator|=(EventHandler::KeyModifier& a, const EventHandler::KeyModifier& b) noexcept
+	{
+		using underlying = std::underlying_type_t<EventHandler::KeyModifier>;
+		auto underlying_a = static_cast<underlying>(a);
+		auto underlying_b = static_cast<underlying>(b);
+		a = static_cast<EventHandler::KeyModifier>(underlying_a | underlying_b);
+		return a;
+	}
 
-EventHandler::KeyModifier& operator&=(EventHandler::KeyModifier& a, const EventHandler::KeyModifier& b) noexcept
-{
-	using underlying = std::underlying_type_t<EventHandler::KeyModifier>;
-	auto underlying_a = static_cast<underlying>(a);
-	auto underlying_b = static_cast<underlying>(b);
-	a = static_cast<EventHandler::KeyModifier>(underlying_a & underlying_b);
-	return a;
-}
+	EventHandler::KeyModifier& operator&=(EventHandler::KeyModifier& a, const EventHandler::KeyModifier& b) noexcept
+	{
+		using underlying = std::underlying_type_t<EventHandler::KeyModifier>;
+		auto underlying_a = static_cast<underlying>(a);
+		auto underlying_b = static_cast<underlying>(b);
+		a = static_cast<EventHandler::KeyModifier>(underlying_a & underlying_b);
+		return a;
+	}
 
-EventHandler::KeyModifier operator|(EventHandler::KeyModifier a, const EventHandler::KeyModifier& b) noexcept
-{
-	NAS2D::operator|=(a, b); //Disambiguate from free function
-	return a;
-}
+	EventHandler::KeyModifier operator|(EventHandler::KeyModifier a, const EventHandler::KeyModifier& b) noexcept
+	{
+		operator|=(a, b);
```

Comment:
Now that the ambiguity has been resolved, this should be changed to just `a |= b`